features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -222,28 +222,57 @@ against unverified tokens.
 
 ## Results
 
-Sign-in confirmation was deployed over several train cycles.
-The feature was turned on at 6-24-2016 for 2% of our users and
-gradually increased to 100%. During the deployment we monitored
-the sign-in success and error rates to ensure that users were
-not negatively impacted.
+Sign-in confirmation was deployed over several
+train cycles. The feature was turned on at 6-24-2016
+for 2% of our users and gradually increased to 100%.
+During the deployment we monitored the sign-in success
+and error rates to ensure that users were not negatively
+impacted.
 
 To help gauge the success of the feature, several dashboards
 were created.
 
 ### Sync sign-in success rate
 
-Pre sign-in confirmation metrics show that we had between a 42% to 52%
-sync sign-in success rate. From [first dashboard](https://app.datadoghq.com/dash/163668/fxa-content---signin-confirmation), with sign-in confirmation, we have maintained this sign-in rate.
+Pre sign-in confirmation metrics show that we had
+between a 42% to 52% sync sign-in success rate.
```

Comment:
I forgot, the `signin` event is further complicated by signin confirmation and the coming signin unblock, and even OAuth permissions which we don't currently show.

Figuring out what `signin.success` means is not an easy task. Does it mean the user has fully completed the signin flow, post email verification? With signin confirmation, we can't measure that using only client side metrics because the client doesn't always know when the user has completed the flow since users sometimes close FxA before verifying their email.